projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.service.ts

```diff
@@ -41,6 +49,18 @@ export class OrderDetailsService {
   }
 
   getOrderDetails(): Observable<Order> {
+    if (this.unifiedInjector) {
+      const serivce = getLastValueSync(
+        this.unifiedInjector.get(OrderDetailsServiceTransitionalToken)
+      );
+      if (serivce) {
+        return serivce.getOrderDetails();
+      }
+      throw Error(
+        'Cannot get order details. Install `@spartacus/order` library which provides required services.'
+      );
```

Comment:
This might not be backwards compatible.  If a project is not using the order lib, things should work.  Let's discuss.

Reply:
Fixed.  Removing the `throw` makes it backwards compatible.